module-4/hashcat/README.md

```diff
@@ -0,0 +1,95 @@
+# My `hashcat` tool
+
+This folder contains the source code for my own implementation of the `hashcat` tool.
+Hashcat can be used to TBD.
```

Comment:
adjust this section to shortly describe your implementation and what it can be used to.